anago

```diff
@@ -763,7 +807,7 @@ get_build_candidate () {
 
   # For master branch alpha builds, ensure we've released the previous .0 first
   if [[ -z "$PARENT_BRANCH" && -n ${RELEASE_VERSION[alpha]} ]]; then
-    latest_official=${BASH_REMATCH[1]}.$((${BASH_REMATCH[2]}-1))
+    latest_official="${BASH_REMATCH[1]}.$((${BASH_REMATCH[2]}-1)).0"
```

Comment:
Is this change related to this PR? Also it seems like lines 815 and 817 below expect the ".0" to be left off. Maybe just add it on line 812?